README.md

```diff
@@ -6,8 +6,6 @@
 
 ### Links and Resources
 
- - [PR](https://github.com/Mariko-401-AdvancedJs/basic-express-server/pulls)
-
  - Simon,Nathan,Matt 
 
```

Comment:
This looks great so far, if you have a chance to come back to this and figure out the image I will regrade for you!